game.py

```diff
@@ -0,0 +1,41 @@
+from monster import Monster
+from hero import Hero
+
+
+def choose_action(monster: Monster, hero: Hero):
+    action = int(
+        input(
+            "enter 1 - for attack ,enter 2 - for level up , enter 3- for heal, enter 4 - for defend    "
+        )
+    )
+    if action == 1:
+        hero.increase_coins(1)
+        hero.attack(monster)
+    elif action == 2:
+        hero.increase_coins(1)
+        hero.level_up()
+    elif action == 3:
+        hero.heal()
+        hero.increase_coins(1)
+    elif action == 4:
+        hero.defend()
+        hero.increase_coins(1)
```

Comment:
increase_coins() shoild be outside the if staments, and then you only need to call it once